### ISSUE TITLE
Update Release Message to Reflect Bazel 6 Conventions

### ISSUE THREAD

**Issue by KetanReddy** [player-ui/rules_player#55]
Current Message looks like 

```
http_archive(
  name = "rules_player",
  strip_prefix = "rules_player-1.0.0",
  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.0.0.tar.gz"],
  sha256 = "d3c305a4bcd39e6c2b21986f1357012f6d5c4aa9a35f30fa487d878537fc851e"
)
```

but what it needs to be is 

```
archive_override(
  module_name = "rules_player",
  strip_prefix = "rules_player-1.0.0",
  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.0.0.tar.gz"],
  integrity = "sha256-08MFpLzTnmwrIZhvE1cBL21cSqmjXzD6SH2HhTf8hR4="
)
```

Key differences are:
- Different macro
- `module_name` vs `name`
- Different sha argument `integrity` vs `sha256`
- Different sha format `<algo>-<digest>` vs `<digest>`. Note that the digest needs to be in the SRI format

**Comment by KetanReddy** [comment]
Docs on SRI -> https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

**Comment by intuit-svc** [comment]
:rocket: Issue was released in [`v1.0.1`](https://github.com/player-ui/rules_player/releases/tag/v1.0.1) :rocket:
